cdap-docs/tracker-manual/source/index.rst

```diff
@@ -18,46 +18,53 @@ Cask Tracker ("Tracker") is a self-service CDAP extension that automatically cap
 :ref:`metadata <metadata-lineage>` and lets you see how data is flowing into and out 
 of datasets, streams, and stream views.
 
-It allows you to perform impact and root-cause analysis, and delivers an audit-trail for
-auditability and compliance. Tracker furnishes access to structured information that
-describes, explains, locates, and makes it easier to retrieve, use, and manage datasets.
+It allows you to perform impact and root-cause analysis, delivers an audit-trail for
+auditability and compliance, and allows you to preview data. Tracker furnishes access to
+structured information that describes, explains, locates, and makes it easier to
+retrieve, use, and manage datasets.
 
-Tracker also allows for the storage of metadata where it can be accessed and indexed. This
-allows it to be is easily searched and provides consistent, high-quality metadata.
+Tracker also allows users to update metadata for datasets and streams. Users can add,
+remove, and update tags or user properties directly in the UI. We also allow users to set
```

Comment:
update tags **and** user properties directly in the UI. **It allows** users to set